angular/src/app/workflows/workflow-list/workflow-list.component.html

```diff
@@ -22,16 +22,19 @@ <h3 class="flex vertical-center">
       </mat-expansion-panel-header>
     </mat-expansion-panel>
     <mat-expansion-panel *ngFor="let run of workflow.runs">
-      <mat-expansion-panel-header>
+      <mat-expansion-panel-header data-se="run">
         <mat-panel-title data-se="run-id">{{ run.run_id }}</mat-panel-title>
         <mat-panel-description data-se="run-state">{{ run.state }}</mat-panel-description>
       </mat-expansion-panel-header>
-      <button mat-button
-              disabled
-              color="primary"
-              class="uppercase">
-        View
-      </button>
+      <a [routerLink]="[workflow.damId, 'views', workflow.viewId, 'runs', run.run_id]">
+        <button mat-button
+                color="primary"
+                class="uppercase"
+                data-se="view-btn"
+        >
```

Comment:
(for future use) Let's keep following the running standards, we have started with Viliam:

1. if there is one property/attribute keep it on a line with a tag, for each new add it on new line
2. `>` should stay on last line, not on a new line

Ideally we would have prettier tool for formatting all of it